build.gradle

```diff
@@ -2564,7 +2568,9 @@ project(':connect:runtime') {
     implementation libs.mavenArtifact
     implementation libs.swaggerAnnotations
```

Comment:
Do we need this to be `implementation` or can it be `swagger` too?

Reply:
I gave this a try but it wasn't successful. We still need the annotations at compile time since they're used to decorate our REST resource classes, like here: https://github.com/apache/kafka/blob/b449b8032edb0e4e6b3213e38a38206a3048c6f5/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java#L108-L114